### DISCUSSION TITLE
Error installing Spatie

### DISCUSSION THREAD

**Discussion by EdTapia** [spatie/laravel-activitylog#978] [Q&A] [answered]
Hello, when installing Spatie v4 in Laravel 8 the following appears:

![image](https://user-images.githubusercontent.com/93604618/139886873-4d666b24-e5ab-4650-9854-85d2520b8c9b.png)

I think it is a versioning problem.
I have laravel 8 and updated PHP to v8.0.12

![image](https://user-images.githubusercontent.com/93604618/139889505-e2cf9c07-8266-4860-8e51-240b6c79eb6a.png)
My composer.json file:
![image](https://user-images.githubusercontent.com/93604618/139890920-4c6c5e53-1eb0-4e33-b340-4e47bfe65e68.png)

from within my application the sentence (PHP v {{PHP_VERSION}} gives me the result v7.1.3

any idea what happens and how to fix it?

**Comment by Gummibeer** [comment]
Seems like your PHP-FPM version isn't the same as your CLI one. I have no idea about AWS and how/where to configure it. But CLI is a whole different binary than the PHP-FPM one.
So you should check how to configure the PHP version for your web-server/image/container.

And one thing: please use markdown code-blocks instead of screenshots if you want to show code. Images aren't accessible and I can't copy anything to try something for example.

**Comment by EdTapia** [comment, marked as answer]
I just solved it by matching the Ubuntu and Apache PHP versions.

$ sudo a2dismod php7.1
$ sudo a2enmod php8.0
$ sudo service apache2 restart

However I was only able to install version 3.17.0 of ActivityLog.
Thank you
